main.go

```diff
@@ -6,6 +6,8 @@ import (
 
 	"github.com/andremedeiros/loon/cmd"
 	"github.com/andremedeiros/loon/internal/ui/color"
+
+	_ "github.com/go-bindata/go-bindata"
```

Comment:
This is so that the module stays, because we require the tool and it kept changing the `go.mod` file without need.